staging/src/k8s.io/apiserver/pkg/apis/audit/register.go

```diff
@@ -49,5 +49,6 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&Policy{},
 		&PolicyList{},
 	)
+	scheme.AddGeneratedDeepCopyFuncs(GetGeneratedDeepCopyFuncs()...)
```

Comment:
Note: this ugly line goes away with phase 2 of static deepcopy https://github.com/kubernetes/kubernetes/pull/48544